cvat-core/src/download.worker.js

```diff
@@ -20,7 +20,6 @@ onmessage = (e) => {
         .catch((error) => {
             postMessage({
                 id: e.data.id,
-                error: error,
```

Comment:
It was redundant information and resulted in an emergence of `DataCloneError: The object could not be cloned`